### ISSUE TITLE
chrome sources doen't updated all the time with ng build -w

### ISSUE THREAD

**Issue by royigoldi** [angular/angular-cli#6826]
### Versions.
windows 7
angular cli 1.1.2

### Repro steps.
working on angular 4 app with cli.
when saving files using webstorm i see the webpack process in the terminal window finshed successfully.
refreash the broswer(F5) with clear cache.

the sources doesn't update with the change and the web behivor is before the save...

**Comment by todrules** [comment]
I'm having the same issue.

Mac OS 10.12.5
Angular CLI 1.1.3

I'm using Webstorm with "safe delete" set to OFF. I notice that after I change a file and save that I still get the same error on recompile, even if the referenced lines and/or files may not even still exist. I just got an error message for a file that I had just deleted. Also, a couple more things to note:

1) The modified date/time IS getting updated for the file outside of Webstorm when updating.
2) If I open the file in another editor, like Atom, I can see the changes I just made. So, it appears that the changes are propagating outside of Webstorm to the file system correctly, but for some reason Webpack is not picking it up.
